jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -263,7 +306,7 @@ private static void setSearchStrategy(String p) {
     private static PageLoadStrategy getPageLoadStrategy(String strategy) {
         switch (strategy.toLowerCase()) {
             case "normal": return NORMAL;
-            case "none": return NONE;
+            case "none": return PageLoadStrategy.NONE;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'PageLoadStrategy.NONE' due to existing static import 'org.openqa.selenium.PageLoadStrategy.NONE'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)